recipes/default.rb

```diff
@@ -18,4 +18,5 @@
   include_recipe 'base2::users'
   include_recipe 'base2::environment'
   include_recipe 'base2::nrpe'
+  include_recipe 'base2::aws_cw_metrics'
```

Comment:
I think we should conditionally include this recipe based on an attribute which can be overridden in the customers cookbook and the default should be to not include it by default. Right now I want this to be an opt-in

Reply:
@aaronwalker 

Updated recipes so it's not in default at all, if metrics are required they can be included manually

```
include_recipe 'base2::aws_cw_metrics
```